Networking/Sources/Networking/Remote/MockNetwork.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  File.swift
+//
+//
+//  Created by marwa on 23/08/2022.
+//
+
+import Foundation
+
+public protocol ReviewsRemoteProtocol {
+    func loadReviewProductData(completion: @escaping (Reviews?, Error?) -> Void)
+}
+
+public class MockNetwork: ReviewsRemoteProtocol {
+
+    public init() {}
+
+    public func loadReviewProductData(completion: @escaping (Reviews?, Error?) -> Void) {
+        var Reviewsproduct: Reviews
+        do {
+            let allProductData = try JSONSerialization.data(withJSONObject: Constants.mockProductReviewsJsonResponse, options: .fragmentsAllowed)
+            Reviewsproduct = try JSONDecoder().decode(Reviews.self, from: allProductData)
+            completion(Reviewsproduct, nil)
+        } catch {
+            completion(nil, error)
+        }
+    }
+}
+
+extension MockNetwork {
+
+    private enum Constants {
+
+        static let mockProductReviewsJsonResponse: [String: Any] = [
```

Comment:
Let's move this json to a file a try to decode it instead of making it hard coded here. You can read about load data from file. It needs only one line of code.